### PR TITLE
Add a ruleset for strasbourgfestival.com

### PR DESCRIPTION
**Pull request by Bischnu**
***Delete section if irrelevant***

### List related PRs if any

- pr link

### List related issues if any

- issue link

***May delete note after reading***

### Note to contributor:
Thank you for contributing! If this is a new ruleset and your first time contributing, congratulations! There will be some tests ran through our continuous integration in Travis. For example, checking for duplicate hosts, problematic hosts, etc. in the ruleset file. You will see this testing begin when you open this pull request.

If this is not your first time and you have open PRs that have not been closed or merged, please revisit them by searching: `https://github.com/EFForg/https-everywhere/issues?q=is%3Aopen+assignee%3A[yourusernamehere]`. If there is something not being addressed by other maintainers or a project lead, link them to the PR with your question. Thank you for your patience.

More helpful filters in Pull Request search:
- Changes requested: 
`is:pr is:open type:pr review:changes_requested author:[your username]`
- Updated from specific date: 
`is:pr updated:>yyyy-mm-dd`
- Merged PRs: 
 `is:pr is:merged author:[your username]`
- Approved PRs: 
`is:pr review:approved author:[your username]`

### CODE REVIEW

File: src/chrome/content/rules/strasbourgfestival.com.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<ruleset name="strasbourgfestival.com">
+
+        <target host="strasbourgfestival.com" />
+
+        <rule from="^http:"
+                to="https:" />
+
+	<test url="http://strasbourgfestival.com/" />
+
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/strasbourgfestival.com.xml:10]:
```suggestion
<ruleset name="strasbourgfestival.com">
	<target host="strasbourgfestival.com" />
	<target host="www.strasbourgfestival.com" />

	<rule from="^http:" to="https:" />
</ruleset>
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

File: src/chrome/content/rules/strasbourgfestival.com.xml (hunk 1 above)
Comment (Bischnu) [reply src/chrome/content/rules/strasbourgfestival.com.xml:10]: Hello,
I am beginner with Git and I do not know how to modify an already-made commit or push.

File: src/chrome/content/rules/strasbourgfestival.com.xml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="strasbourgfestival.com">
+    <target host="strasbourgfestival.com" />
+    <target host="www.strasbourgfestival.com" />
+
+  <rule from="^http:" to="https:" />
```
Comment (pipboy96) [inline src/chrome/content/rules/strasbourgfestival.com.xml:5]:
```suggestion
	<target host="strasbourgfestival.com" />
	<target host="www.strasbourgfestival.com" />

	<rule from="^http:" to="https:" />
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]
Please use tab characters instead of spaces

**Comment by pipboy96** [comment]
Don't worry about editing the commits, they will be merged as one commit anyway.

**Review by pipboy96 — APPROVED** [review]

**Pull request merged**